### PR TITLE
Wikimedia cloud

### PR DESCRIPTION
**Pull request by MoriyaNadav**

### CODE REVIEW

File: services/a.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from datetime import datetime
```
Comment (yonathan06) [inline services/a.py]: is this file used anywhere?

File: services/initdaily.sh [hunk 2]
```diff
@@ -1,2 +1,5 @@
 #!/bin/bash
-python3 $HOME/cassandra-GLAM-tools/services/daily.py  -e production
\ No newline at end of file
+python3 $HOME/cassandra-GLAM-tools/services/daily.py  -e production
+
+
+curl -X POST http://serviceapp.glamwikidashboard.eqiad1.wikimedia.cloud:8080/ -H 'Content-Type: application/json' -d '{"login":"my_login","password":"my_password"}'
```
Comment (yonathan06) [inline services/initdaily.sh]: why do we need it in initdatily.sh?

Comment (MoriyaNadav) [reply services/initdaily.sh]: Sorry' I forgot to delete this line :)

File: services/a.py (hunk 1 above)
Comment (MoriyaNadav) [reply services/a.py]: Sorry' I forgot to delete this file (I created it for testing :)

**Review by yonathan06 — APPROVED** [review]

**Comment by MoriyaNadav** [comment]
I fixed some issue and create new PR

**Pull request closed without merging**